openapi.yml

```diff
@@ -237,63 +237,24 @@ components:
       required:
         - hasAdminPolicy
     AdminPolicyAccessTemplate:
-      description: 'Provides the template of access settings that is copied to the items goverend by an AdminPolicy. This is almost the same as DROAccess, but it provides no defaults and has no embargo.'
+      description: 'Provides the template of access settings that is copied to the items governed by an AdminPolicy. This is almost the same as DROAccess, but it provides no defaults and has no embargo.'
       type: object
       additionalProperties: false
+      oneOf:
+        - $ref: '#/components/schemas/DarkAccess'
+        - $ref: '#/components/schemas/CitationOnlyAccess'
+        - $ref: '#/components/schemas/ControlledDigitalLendingAccess'
+        - $ref: '#/components/schemas/LocationBasedAccess'
+        - $ref: '#/components/schemas/LocationBasedDownloadAccess'
+        - $ref: '#/components/schemas/StanfordAccess'
+        - $ref: '#/components/schemas/WorldAccess'
```

Comment:
This is how we add access template validation, by leaning on schemas we already have on hand.